src/components/Overview/Overview.react.js

```diff
@@ -19,6 +19,12 @@ import susiSkill from '../../images/susi_skill.png';
 import susiTestGif from '../../images/susi-test.gif';
 import { Link } from 'react-router-dom';
 import React, { Component } from 'react';
+import "animate.css/animate.min.css";
+import ScrollAnimation from 'react-animate-on-scroll';
+<link rel="stylesheet" href="bower_components/aos/dist/aos.css" />
```

Comment:
Use `import`?

Reply:
Yep, I've used import... Sorry, I didn't get what you're trying to say..

Reply:
Use `import` rather than link.